packages/flutter_test/lib/src/controller.dart

```diff
@@ -1243,6 +1458,8 @@ abstract class WidgetController {
   /// the widget's render object has been laid out at least once.
   Rect getRect(Finder finder) => Rect.fromPoints(getTopLeft(finder), getBottomRight(finder));
 
+  // TODO(pdblasi-google): Deprecate this and point references to semantics.find.
```

Comment:
If we do choose to deprecate, can you create an issue to track it and link it here?

Reply:
Good call. The issue is #112670 if you're curious. Also added to the comment.